### PR TITLE
feat(select): select component MVP

### PR DESCRIPTION
**Pull request by nik-kor**
hey, guys, it brings really minimum value. So a lot of details and use cases are missed.

### CODE REVIEW

File: examples/components/icons/icons.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const React = require('react');
 
 const Icon = require('../../../src').Icon;
-require('../../design/icon/icon.styl');
```
Comment (nik-kor) [inline examples/components/icons/icons.jsx]: I didn't get why - but bem-loader tries to load design/icon/icon.styl file and couldn't do this

File: examples/package.json [hunk 2]
```diff
@@ -13,8 +13,8 @@
     "bem-components": "git+ssh://git@github.com/awinogradov/bem-components.git#v2",
     "bem-core": "git+ssh://git@github.com/bem/bem-core.git#v2",
     "bem-xjst": "git+ssh://git@github.com:awinogradov/bem-xjst.git#feature/bemreact",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7",
+    "react": "^15.1.0",
+    "react-dom": "^15.1.0",
```
Comment (nik-kor) [inline examples/package.json]: There was something weird with old react. React@14 created two nodes with the same reactid. Looks like a bug. But it  works with react@15

File: package.json [hunk 3]
```diff
@@ -10,21 +10,23 @@
...
     "bem-components": "git+ssh://git@github.com/awinogradov/bem-components.git#v2",
     "bem-core": "git+ssh://git@github.com/bem/bem-core.git#v2",
+    "bem-loader": "^0.5.0",
     "bem-naming": "^1.0.1",
     "bem-xjst": "git+ssh://git@github.com:awinogradov/bem-xjst.git#feature/bemreact",
-    "bem-loader": "^0.5.0",
     "browserify": "^13.0.0",
+    "fs-extra": "^0.26.7",
     "param-case": "^1.1.2",
+    "react": "^15.1.0",
```
Comment (nik-kor) [inline package.json]: I didn't get why do react is a dep here?

File: src/components/menu/menu.js [hunk 4]
```diff
@@ -13,6 +13,8 @@ class Menu extends Control {
             attrs: {
                 onMouseEnter: this._onMouseEnter.bind(this),
                 onMouseLeave: this._onMouseLeave.bind(this),
+                // TODO
+                onKeyDown: this.handleKeyDown.bind(this),
```
Comment (nik-kor) [inline src/components/menu/menu.js]: didn't work for me and I had thought that it wasn't mvp-feature really

File: examples/package.json (hunk 2 above)
Comment (Guria) [inline examples/package.json]: I bet there was a few instances of react used.

File: package.json (hunk 3 above)
Comment (Guria) [inline package.json]: it should be in peer deps section, of course

Comment (Guria) [inline package.json]: and most of other deps are just dev deps here

File: examples/package.json (hunk 2 above)
Comment (nik-kor) [inline examples/package.json]: hm. I'll check it

**Comment by awinogradov** [comment]
@nik-kor are you ready to merge?)

**Comment by nik-kor** [comment]
@awinogradov not so fast)

I have to fix comments from @Guria

**Comment by awinogradov** [comment]
👍

**Pull request merged**